draft-ietf-quic-transport.md

```diff
@@ -924,11 +924,12 @@ selected by the client, both to ensure correct routing toward the client and to
 allow the client to validate that the packet is in response to an Initial
 packet.
 
-A zero-length connection ID MAY be used when the connection ID is not needed for
-routing and the address/port tuple of packets is sufficient to identify a
-connection. An endpoint whose peer has selected a zero-length connection ID MUST
-continue to use a zero-length connection ID for the lifetime of the connection
-and MUST NOT send packets from any other local address.
+A zero-length connection ID can be used when a connection ID is not needed
+to route to the correct endpoint. However, multiplexing connections on the same
+local IP address and port while using zero-length connection IDs will cause
+failures in the presence of connection migration, NAT rebinding, and client port
+reuse; and therefore MUST NOT be done unless an endpoint is certain that those
+protocol features are not in use.
```

Comment:
If I were to add commentary here, I would say "While this usage model appears to be substantially similar to common TCP connection idioms, the realities of UDP deployment mean that relying on peer addressing information for identifying connections can be significantly less reliable in UDP than TCP."

Reply:
(Replying self, bad look, sorry.) Is this (^^) something that the applicability draft might want to say? @mirjak, @britram, WDYT?

Reply:
Both sound good to me, I agree that this is a good point to make, and the applicability draft sounds like a good home